samples/WorkerBindingSamples/Table/TableInputBindingSamples.cs

```diff
@@ -0,0 +1,123 @@
+            [TableInput("TableName", "{partitionKey}", "{rowKey}")] TableEntity table)
+
+        {
+            table.TryGetValue("Text", out var text);
+            var response =  req.CreateResponse(HttpStatusCode.OK);
+            await response.WriteStringAsync(text?.ToString() ?? "");
+            return response;
+        }
+
+        [Function(nameof(ReadTableDataFunctionWithFilter))]
+        public async Task<HttpResponseData> ReadTableDataFunctionWithFilter(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+            [TableInput("TableName", "My Partition", 2, Filter = "RowKey ne 'value'")] IEnumerable<TableEntity> table)
+
+        {
+            List<string> tableList = new();
+            var response = req.CreateResponse(HttpStatusCode.OK);
+
+            foreach (TableEntity tableEntity in table)
+            {
+                tableEntity.TryGetValue("Text", out var text);
+                _logger.LogInformation("Value of text: " + text);
+                tableList.Add(text?.ToString() ?? "");
+            }
+
+            await response.WriteStringAsync(string.Join(",", tableList));
+            return response;
+        }
+
+
+        [Function(nameof(EnumerableFunction))]
+        public async Task<HttpResponseData> EnumerableFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = "items/{partitionKey}")] HttpRequestData req,
+            [TableInput("TableName", "{partitionKey}")] IEnumerable<TableEntity> tables)
+
+        {
+            var response = req.CreateResponse(HttpStatusCode.OK);
+            List<string> tableList = new();
+
+            foreach (TableEntity tableEntity in tables)
+            {
+                tableEntity.TryGetValue("Text", out var text);
+                _logger.LogInformation("Value of text: " + text);
+                tableList.Add((text?.ToString()) ?? "");
+                
+            }
+
+            await response.WriteStringAsync(string.Join(",", tableList));
+            return response;
+        }
+
+        [Function(nameof(PocoFunction))]
+        public async Task<HttpResponseData> PocoFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get","post", Route = null)] HttpRequestData req,
+            [TableInput("TableName")] IEnumerable<MyEntity> entities,
+            FunctionContext executionContext)
+        {
+            var logger = executionContext.GetLogger(nameof(PocoFunction));
+
+            var response = req.CreateResponse(HttpStatusCode.OK);
+            List<string> entityList = new();
+
+            foreach (MyEntity entity in entities)
+            {
+                logger.LogInformation($"Text: {entity.Text}");
+                entityList.Add((entity.Text ?? "").ToString());
+            }
+
+            await response.WriteStringAsync(string.Join(",", entityList));
+            return response;
```

Comment:
```suggestion
            return await response.WriteStringAsync(string.Join(",", entityList));
```

Reply:
This needs to be seperate lines otherwise it doesn't compile. WriteStringAsync is void so we need to return response itself after the call has been done